rules.adoc

```diff
@@ -775,7 +775,7 @@ ball.
 [[regulations-inference-in-lightweight]]
 ==== Infrared interference in Lightweight
 
-In Lightweight, the robot must not emit infrared light.
+Components designed to emit IR (e.g. IR time-of-flight LiDAR sensors) are not allowed and must be covered or removed if present at all. Any device designed to emit light of any wavelength must be mounted so that light is directed perpendicular to the playing field or be covered/removed.
```

Comment:
I'd like to include a couple more examples (e.g. IR distance sensors, LiDAR sensors, IR leds/lasers of any kind) because the ever-popular SHARP IR Distance sensors use traingulation, some sensors use correlation of modulated light etc. ToF is requires sub-nanosecond precision and is therefore not common. I am in favor of the phrasing "designed to emit IR" though.

Reply:
I'm concerned about the length and long-term relevance of adding more examples; I only included the one b/c that was the cause of all of this. How about adding "Please search the forum and post any questions that are not already clarified there."

Reply:
Length I somewhat understand but longevity I can speak to: IR sensors haven't really changed since my first RCJ Soccer year of 2009 (iirc). LiDARs weren't common and have started rotating but everything else is as relevant as ever esp. for teams that are coming from Rescue where active IR sensors continue to be useful.